core/controllers/email_dashboard.py

```diff
@@ -87,27 +93,17 @@ def post(self):
         kwargs = {key: data[key] for key in data if data[key] is not None}
         self._validate(kwargs)
 
-        query_id = user_query_services.save_new_query_model(
+        user_query = user_query_services.save_new_user_query(
```

Comment:
As discussed in a previous comment, could you return the ID only?

Reply:
This was fixed already. We collided when I was pushing stuff and you reviewing.